.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: python
 python:
-  - "2.7"
   - "3.4"
   - "3.5"
   - "3.6"
 env: TEST_SUITE=test-main
 matrix:
   include:
+  - python: "2.7"
```

Comment:
Do we actually need this?  If so, I would clarify the commit message.

Reply:
Looks correct to me.

Reply:
(But I think the commit message could benefit here, yeah)